config/rewrites.d/docs.rackspace.com.json

```diff
@@ -125,6 +197,78 @@
 			"toHostname": "developer.rackspace.com",
 			"rewrite": false,
 			"status": 301
+		},
+		{
+			"description": "cloud networks",
+			"from": "^/networks/api/v2/?(.*)$",
+			"to": "/docs/cloud-networks/v2/",
+			"toProtocol": "https",
+			"toHostname": "developer.rackspace.com",
+			"rewrite": false,
+			"status": 301
+		},
+		{
+			"description": "orchestration templates user guide",
+			"from": "^/orchestration/api/v1/orchestration-templates-devguide",
+			"to": "/docs/user-guides/orchestration/",
+			"toProtocol": "https",
+			"toHostname": "developer.rackspace.com",
+			"rewrite": false,
+			"status": 301
+		},
+		{
+			"description": "orchestration API",
+			"from": "^/orchestration/api/v1/?(.*)$",
+			"to": "/docs/cloud-orchestration/v1/",
+			"toProtocol": "https",
+			"toHostname": "developer.rackspace.com",
+			"rewrite": false,
+			"status": 301
+		},
+		{
+			"description": "Rackconnect",
+			"from": "^/rackconnect/api/v3/?(.*)$",
+			"to": "http://docs.rcv3.apiary.io/",
+			"toProtocol": "https",
+			"toHostname": "developer.rackspace.com",
+			"rewrite": false,
+			"status": 301
+		},
+		{
+			"description": "cloud servers legacy",
+			"from": "^/servers/api/v1.0/?(.*)$",
+			"to": "/docs/cloud-servers/v2/",
+			"toProtocol": "https",
+			"toHostname": "developer.rackspace.com",
+			"rewrite": false,
+			"status": 301
+		},
+		{
+			"description": "cloud servers",
+			"from": "^/servers/api/v2/?(.*)$",
+			"to": "/docs/#docs-cloud-servers-legacy",
+			"toProtocol": "https",
+			"toHostname": "developer.rackspace.com",
+			"rewrite": false,
+			"status": 301
+		},
+		{
+			"description": "Rackspace Private Cloud Operator and User Guide, v10",
```

Comment:
should we redirect to the docs landing page in case any old links are lying around in Google search or bookmark lists?